components/tikv_util/src/security.rs

```diff
@@ -144,6 +186,42 @@ impl SecurityManager {
     }
 }
 
+struct Fetcher {
+    certs: Arc<RwLock<Certs>>,
+    last_modified: Arc<RwLock<SystemTime>>,
+    cfg: Arc<SecurityConfig>,
+}
+
+impl ServerCredentialsFetcher for Fetcher {
+    fn fetch(&self) -> Result<Option<ServerCredentialsBuilder>, Box<dyn Error>> {
```

Comment:
What if the certificate is outdated but there is no new connections to TiKV, then the cerfiticates are never reloaded? It will affect the client building new connections to other server or connecting to PD.

Reply:
This situation does occur.

Reply:
fixed